packages/plugin-infra/src/web3/Context.tsx

```diff
@@ -81,11 +81,17 @@ export function usePluginIDContext() {
 
 const PluginsWeb3StateContext = createContainer(usePluginsWeb3State)
 
-export const usePluginsWeb3StateContext = PluginsWeb3StateContext.useContainer
+// usePluginWeb3Context is used to provide a set of API to interact with Web3.
+export function usePluginWeb3Context(expectedPluginID?: string) {
```

Comment:
`usePluginWeb3Context()` functions this same as `useWeb3State()`, but more readable.